server/routes/journeys/update-review/close-csip/controller.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response } from 'express'
+import { PatchReviewController } from '../../base/patchReviewController'
+
+export class UpdateCloseCsipController extends PatchReviewController {
+  GET = async (req: Request, res: Response) => {
+    req.journeyData.isUpdate = true
+    res.render('record-review/close-csip/view', {
+      closeCsip: res.locals.formResponses?.['closeCsip'] ?? req.journeyData.review?.outcome,
+      isUpdate: true,
+      backUrl: `outcome`,
+      recordUuid: req.journeyData.csipRecord!.recordUuid,
+      openIdentifiedNeeds: (req.journeyData.csipRecord?.plan?.identifiedNeeds || []).filter(o => !o.closedDate).length,
```

Comment:
probably not needed

Reply:
It's needed to display the warning text of "X open identified needs"